js/repl/UriUtils.js

```diff
@@ -4,20 +4,22 @@ import LZString from "lz-string";
 
 import type { PersistedState } from "./types";
 
-const URL_KEYS = [
-  "babili",
-  "browsers",
-  "build",
-  "builtIns",
-  "code",
-  "debug",
-  "circleciRepo",
-  "evaluate",
-  "lineWrap",
-  "presets",
-  "targets",
-  "version",
-];
+const URL_DEFAULTS = {
+  babili: false,
+  browsers: "",
+  build: "",
+  builtIns: false,
+  circleciRepo: "",
+  code: "",
+  debug: false,
+  evaluate: false,
+  lineWrap: true,
```

Comment:
This looks okay to me, but I'd like @bvaughn to take a look too (if he gets time to). It seems like [replUtils.js](https://github.com/babel/website/blob/f34ae222d3f7edf70711e1c849f13418da95b4cb/js/repl/replUtils.js#L52) is also doing some defaulting, so I wonder if we could unify the two such that they both share the same set of defaults.

Reply:
That make sense. I'll make it generic and update the PR.